docs/REST API/rest-api.md

```diff
@@ -6712,8 +6712,8 @@ None.
 - [List team policies](#list-team-policies)
 - [Count policies](#count-policies)
 - [Count team policies](#count-team-policies)
-- [Get policy](#get-policy)
-- [Get team policy](#get-team-policy)
+- [Get policy by ID](#get-policy-by-id)
+- [Get team policy by ID](#get-team-policy-by-id)
```

Comment:
I think these naming changes are also an unintentional result of switching the branch (we recently removed "by ID" on `main`).

But hey, it's not 200+ files worth of changes so I'd call it a win! 😅 
```suggestion
- [Get policy](#get-policy)
- [Get team policy](#get-team-policy)
```